src/main/java/com/example/demo/service/implementation/ProjectServiceImplementation.java

```diff
@@ -68,35 +85,115 @@ public ResponseEntity<ProjectResponseDto> updateProject(Long id, ProjectRequestD
 	public ResponseEntity<ProjectResponseDto> removeProject(Long id) {
 		// TODO Auto-generated method stub
 		Project project=projectdao.findById(id).orElseThrow(()->new ResourceNotFoundException("No any Project registered with this id ::"+id));
-	project.getEmployee().forEach(e->project.removeEmployee(e));
-	project.getDepartment().forEach(d->d.removeProject(project));
-	projectdao.deleteById(id);
+//	project.getEmployee().forEach(e->project.removeEmployee(e));
+		List<Employee>eList=project.getEmployee();
```

Comment:
if project.getEmployee() returns the list of employees, then it would be advised to have "getEmployees()' instead of "getEmployee()"

Reply:
I agree that using getEmployees() would be more appropriate and intuitive since it returns a list of employees. I will update the method name accordingly to ensure clarity and consistency.